misc/install.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+ENIGMA_NODE_VERSION=4.4
+ENIGMA_INSTALL_DIR=$HOME/enigma-bbs
```

Comment:
Suggest that the entire script is wrapped in `{` and `}`. This will ensure the whole script is complete.